tests/baselines/reference/api/typescript.d.ts

```diff
@@ -10659,6 +10701,7 @@ declare namespace ts {
         commentSelection(fileName: string, textRange: TextRange): TextChange[];
         uncommentSelection(fileName: string, textRange: TextRange): TextChange[];
         getSupportedCodeFixes(fileName?: string): readonly string[];
+        mapCode(mappings: MapCodeDocumentMapping[], formatOptions: FormatCodeSettings, preferences: UserPreferences, updates?: FileTextChanges[]): FileTextChanges[];
```

Comment:
These should be internal for now since this is not completely hashed out from the looks of it
Everything except protocol changes should be internal